contributing/code/maintenance.rst

```diff
@@ -0,0 +1,58 @@
+Maintenance
+===========
+
+During the lifetime of a minor version, new releases (patch versions) are
+published on a monthly basis. This document describes the boundaries of
+acceptable changes.
+
```

Comment:
I don't think it adds much value. It also conveys the idea that as long as risk is low, new features could be added.